eland/ml/ml_model.py

```diff
@@ -269,6 +269,9 @@ def import_model(
 
         model: An instance of a supported python model. We support the following model types:
             - sklearn.tree.DecisionTreeClassifier
+                - NOTE: When calculating the probabilities of a given classification label, Elasticsearch utilizes
+                        softMax. SKLearn instead normalizes the results. This may result in different probabilities
+                        but the overall predictions will still align.
```

Comment:
@tveasey can you think of a way of adjusting the raw leaf values so that when we take the softmax it ends up being the same as normalizing the raw values?

Reply:
Well, `log` could work if I just choose some large negative number for when `n` is 0. I will experiment.